compiler/rustc_symbol_mangling/src/typeid.rs

```diff
@@ -1,42 +1,67 @@
-// For more information about type metadata and type metadata identifiers for cross-language LLVM
-// CFI support, see Type metadata in the design document in the tracking issue #89653.
+// Type metadata identifiers for LLVM Control Flow Integrity (CFI) and cross-language LLVM CFI
+// support.
+//
+// For more information about LLVM CFI and cross-language LLVM CFI support for the Rust compiler,
+// see design document in the tracking issue #89653.
 
+use bitflags::bitflags;
 use rustc_middle::ty::{FnSig, Ty, TyCtxt};
 use rustc_target::abi::call::FnAbi;
 use std::hash::Hasher;
 use twox_hash::XxHash64;
 
+bitflags! {
+    /// Options for typeid_for_fnabi and typeid_for_fnsig.
+    pub struct TypeIdOptions: u32 {
+        const NO_OPTIONS = 0;
```

Comment:
You can use `TypeIdOptions::empty()` for this.

Reply:
Done.